content/docs/reference/glossary.md

```diff
@@ -24,6 +24,10 @@ A unique identifier used to authenticate a user or a calling program to an API.
 
 <a id="branches-coming-soon/"></a>
 
+### Availability checker
```

Comment:
besides the check_availability operation we have several more operations that a user can see in the console UI in the operations log:

- apply_config
- create_branch
- create_timeline
- delete_tenant
- delete_timeline
- replace_safekeeper
- start_compute
- stop_compute
- suspend_compute

What is the plan for those?

Reply:
This work is tracked by:
https://github.com/neondatabase/website/issues/170